reference/recency-surveillance.v1.json

```diff
@@ -59,150 +59,155 @@
+                      "serial_number": {
+                        "description": "HTS number. This should be same number used in HTS blood samples sent to UVRI",
                         "type": "string",
-                        "enum": [
-                          "No",
-                          "Yes"
-                        ]
+                        "default": "text"
                       },
-                      "referred_to_tb_services": {
+                      "visit_date": {
+                        "description": "Must be YYYY-MM-DD format",
                         "type": "string",
-                        "enum": [
-                          "No",
-                          "Yes"
-                        ]
+                        "default": "yyyy-MM-dd ",
+                        "format": "date"
                       },
-                      "presumtive_tb": {
+                      "health_unit_name": {
+                        "description": "Name of health facility",
                         "type": "string",
-                        "enum": [
-                          "No",
-                          "Yes",
-                          "T.B Follow-up"
-                        ]
+                        "default": "text"
                       },
-                      "couple_result": {
+                      "health_unit_sub_county": {
+                        "description": "Name of the subcounty of the client’s address. If not available, name of subcounty the the health facility is located ",
                         "type": "string",
-                        "enum": [
-                          "DISCORDANT COUPLE",
-                          "Cordant",
-                          "Concordant Negative",
-                          "Concordant Positive"
-                        ]
+                        "default": "text"
                       },
-                      "couple_result_received": {
+                      "health_unit_district": {
+                        "description": "Name of the district of the client’s address. If not available, name of district the health facility is located",
                         "type": "string",
-                        "enum": [
-                          "No",
-                          "Yes"
-                        ]
+                        "default": "text"
                       },
-                      "individual_result_received": {
+                      "sex": {
+                        "description": "Gender of client",
                         "type": "string",
                         "enum": [
-                          "No",
-                          "Yes"
+                          "M",
+                          "F"
                         ]
                       },
-                      "recent_hiv_test_result": {
+                      "date_of_birth": {
+                        "description": "Must be YYYY-MM-DD format",
```

Comment:
@ODORA0 update the description to match the default format and also an example date to illustrate

Reply:
@ssmusoke Done, but what did you mean by the date when the client was born?